tests/common/fixtures/advanced_reboot.py

```diff
@@ -390,7 +390,6 @@ def __setupTestbed(self):
 
         logger.info('Copy ARP responder to the PTF container  {}'.format(self.ptfhost.hostname))
         self.ptfhost.copy(src='scripts/arp_responder.py', dest='/opt')
-        self.ptfhost.copy(src='scripts/dual_tor_sniffer.py', dest="/root/ptftests/advanced_reboot_sniffer.py")
         # Replace fast-reboot script
```

Comment:
The `dual_tor_sniffer.py` can be deleted now?

Reply:
Hi @bingwang-ms - it used in dual-tor tests, but somehow it was used also there with different name.